src/routes/Settings.tsx

```diff
@@ -75,13 +77,15 @@ export const SettingsRoute: FC = () => {
 
   useMemo(() => {
     (async () => {
-      const response = await getRootHypermediaLinks(
-        Constants.DEFAULT_AUTH_OPTIONS.hostname,
```

Comment:
perhaps this should get the RootHypermediaLinks by conditionally pass the hostname (eg: GitHub ?? EnterpriseAccounts)

Reply:
I considered addressing this, but it would introduce the concept of a primary account, which could complicate the implementation. Therefore, I opted for the simplest solution.

Reply:
@afonsojramos - thoughts on whether the scopes check is still required?  A few months back I did do a lot of testing locally and couldn't find a scenario where I didn't have permissions to make the necessary API calls for detailed notifications (previously called colors).  Removing this would simplify the above convo

Reply:
Sorry for the late reply, had a couple of intense days

Reply:
I agree

I think we're pretty well covered with:
* dedicated error page for Missing Scopes
* Settings tooltips explaining whats required
* new login flow via `Generate a PAT` selects all the required scopes

I'll raise a PR shortly to simplify this

Reply:
https://github.com/gitify-app/gitify/pull/1127